apps/designer/app/dashboard/project-card.tsx

```diff
@@ -0,0 +1,292 @@
+    if (fetcher.data?.errors) {
+      toast.error(fetcher.data.errors);
+    }
+  }, [fetcher.data]);
+
+  const handleKeyDown = (event: KeyboardEvent<HTMLElement>) => {
+    const elements: Array<HTMLElement> = Array.from(
+      event.currentTarget.querySelectorAll(`[tabIndex='-1']`)
+    );
+    const currentIndex = elements.indexOf(
+      document.activeElement as HTMLElement
+    );
+    switch (event.key) {
+      case "Enter": {
+        designerLinkRef.current?.click();
+        break;
+      }
+      case "ArrowUp":
+      case "ArrowRight": {
+        const nextElement = elements.at(currentIndex + 1) ?? elements[0];
+        nextElement?.focus();
+        break;
+      }
+      case "ArrowDown":
+      case "ArrowLeft": {
+        const nextElement = elements.at(currentIndex - 1) ?? elements[0];
+        nextElement?.focus();
+        break;
+      }
+    }
+  };
+
+  const handleDelete = (projectId: string) => {
+    fetcher.submit(
+      { projectId },
+      { method: "delete", action: "/dashboard/projects/delete" }
+    );
+  };
+
+  const handleRename = (projectId: string) => {
+    // @todo replace with the new dialog UI, waiting for dialog component
+    const title = prompt();
+    // User has aborted
+    if (title === null) {
+      return;
+    }
+    fetcher.submit(
+      { projectId, title },
+      { method: "post", action: "/dashboard/projects/rename" }
+    );
+  };
+
+  const handleDuplicate = (projectId: string) => {
+    fetcher.submit(
+      { projectId },
+      { method: "post", action: "/dashboard/projects/duplicate" }
+    );
+  };
+
+  return {
+    designerLinkRef,
+    handleKeyDown,
+    handleDelete,
+    handleRename,
+    handleDuplicate,
+  };
+};
+
+type ProjectCardProps = DashboardProject;
+
+export const ProjectCard = ({
+  id,
+  title,
+  domain,
+  isPublished,
+}: ProjectCardProps) => {
+  const {
+    designerLinkRef,
+    handleKeyDown,
+    handleDelete,
+    handleRename,
+    handleDuplicate,
+  } = useProjectCard();
+  return (
+    <Flex
+      direction="column"
+      shrink={false}
+      as="article"
+      className={projectCardContainerStyle()}
+      tabIndex={0}
+      onKeyDown={handleKeyDown}
+    >
+      <Flex
+        grow
+        align="center"
+        as={RemixLink}
+        ref={designerLinkRef}
+        to={designerPath({ projectId: id })}
+        className={projectNameAvatarStyle()}
+        tabIndex={-1}
+      >
+        {getAbbreviation(title)}
+      </Flex>
+      <Flex
+        justify="between"
+        shrink={false}
+        gap="1"
+        className={projectCardFooterStyle()}
+      >
+        <Flex direction="column" justify="around">
+          <Text variant="title" truncate>
+            {title}
+          </Text>
+          {isPublished ? (
+            <PublishedLink domain={domain} tabIndex={-1} />
+          ) : (
+            <Text color="hint">Not Published</Text>
+          )}
+        </Flex>
+        {isFeatureEnabled("dashboard2") && (
```

Comment:
> isFeatureEnabled("dashboard2")

Do we still need this?

Reply:
Ah, I thought dashboard2 was used to hide new dashboard

Reply:
since I added support for most items, I will just hide the share item behind the feature flag for now